packages/botonic-cli/src/botonic-api-service.ts

```diff
@@ -1,4 +1,10 @@
-import axios, { AxiosPromise, AxiosResponse } from 'axios'
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
why is complaining here eslint?

Reply:
This is to avoid seeing in many lines (e.g. when creating headers) that the attributes have to be with camelCase.